Update sql-agent-extension-automatic-registration-all-vms.md (#8244)
MicrosoftDocs/sql-docs: azure-sql/virtual-machines/windows/sql-agent-extension-automatic-registration-all-vms.md

Once automatic registration is enabled for a subscription, all current and future VMs that have SQL Server installed will be registered with the SQL IaaS Agent extension **in lightweight mode without downtime, and without restarting the SQL Server service**. You still need to [manually upgrade to full manageability mode](sql-agent-extension-manually-register-single-vm.md#upgrade-to-full) to take advantage of the full feature set. The license type automatically defaults to that of the VM image. If you use a pay-as-you-go image for your VM, then your license type will be `PAYG`, otherwise your license type will be `AHUB` by default. 

> [!NOTE]

> > There might be scenarios where user would have deployed a SQL VM which has IaaS extension preinstalled. Sometimes its in a failure state or sometimes they will remove it. Automatic registration will check the registry to see if SQL Server is installed on a VM and if extension is not there then it would register it in lightweight mode.

Reviewer: ```suggestion
> If you deployed a SQL Server VM with the IaaS extension preinstalled, and the extension is in a failure state or it was removed, automatic registration will check the registry to see if SQL Server is installed on a VM, and register it in lightweight mode.
```

Reviewer 2: ```suggestion
> If you deployed a SQL Server VM with a marketplace image which has the SQL IaaS agent extension preinstalled, and the extension is in a failure state or it was removed, automatic registration will check the registry to see if SQL Server is installed on the VM and register it in lightweight mode.
```
